docs/topics/sand/sand-token.md

```diff
@@ -0,0 +1,46 @@
+---
+breaks: false
+
+description: Sand contract description
+
+---
+
+# [Sand Smart contract](https://github.com/thesandboxgame/sandbox-smart-contracts/blob/master/src/solc_0.5/Sand.sol)
+
+## Introduction
+
+This contract implements an [ERC20](https://eips.ethereum.org/EIPS/eip-20) compatible smart contract that is used to
+represent the SAND token. The SAND token will be used for:
+
+- Trading Assets among players
+- Fee for minting Assets
+- Staking in our curation / moderation system
+- Paying for meta-transactions
+- Voting decisions
+
```

Comment:
Add a sentence that suggests to the user more info on SAND, something like
"Learn more about [SAND](https://sandboxgame.gitbook.io/the-sandbox/sand/what-is-sand)"